components/brave_wallet_ui/components/desktop/portfolio-transaction-item/index.tsx

```diff
@@ -322,9 +323,16 @@ const PortfolioTransactionItem = (props: Props) => {
             <CoinsIcon />
           </CoinsButton>
         </TransactionFeesTooltip>
-        <MoreButton onClick={onShowTransactionPopup}>
-          <MoreIcon />
-        </MoreButton>
+        {transactionDetails.status !== TransactionStatus.Rejected &&
```

Comment:
Instead of having multiple conditionals here,
We could do something like this? 😎

```
{transactionDetails.status !== TransactionStatus.Rejected ? (
          <MoreButton onClick={onShowTransactionPopup}>
            <MoreIcon />
          </MoreButton>
        ) : (
          <RejectedTransactionSpacer />
        )
}
```

We can also self close `RejectedTransactionSpacer` component like so since it has no children.
``<RejectedTransactionSpacer />``

Reply:
Thank you, let me make changes